Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -371,21 +405,29 @@ extension ChatChannel {
         }
         
         let fetchMessages: () -> [ChatMessage] = {
-            MessageDTO
+            let limit = dto.managedObjectContext?.localCachingSettings?.chatChannel.latestMessagesLimit ?? 25
```

Comment:
We should not be accessing this properties from here, but instead pass this as parameter

Reply:
agreed, this is just keeping the same behavior to avoid adding more refactoring work in scope (there's already plenty of work in this PR :))